src/main/java/dto/trades.java

```diff
@@ -11,6 +11,6 @@ class trades {
     private double price;
     private int size;
     private String timestamp;
-    private String condition[];
+    private String[] condition;
```

Comment:
Changed Array declarations from C-style to be Java style declarations

Reply:
Covered in a previous code review.